.github/workflows/build-test.yaml

```diff
@@ -53,7 +59,15 @@ jobs:
         run: kubectl --kubeconfig=$GITHUB_WORKSPACE/.kubeconfig delete ns --ignore-not-found automated-$GITHUB_RUN_ID
 
       - name: install qa app
-        run: ./bin/kots --kubeconfig=$GITHUB_WORKSPACE/.kubeconfig install qakots/github-actions --port-forward=false -n automated-$GITHUB_RUN_ID --shared-password password
+        run: |
+          ./bin/kots --kubeconfig=$GITHUB_WORKSPACE/.kubeconfig \
+          install qakots/github-actions \
+          --port-forward=false \
+          -n automated-$GITHUB_RUN_ID \
+          --shared-password password \
+          --kotsadm-registry ttl.sh \
+          --kotsadm-namespace automated-$GITHUB_RUN_ID \
+          --kotsadm-tag 2h
```

Comment:
this is not unique. wont prs step on eachothers toes?

Reply:
wait. i dont understand how UUID comes into play here

Reply:
`--kotsadm-namespace` is

Reply:
yeah, UUID should solve that

Reply:
--kotsadm-namespace takes care of that